.gitlab-ci.yml

```diff
@@ -171,19 +165,29 @@ examples-clippy-wasm:
 examples-contract-build:
   stage:                           examples
   <<:                              *docker-env
+  dependencies:
+    - build-cli
   script:
+    - ls -la
+    - ls ./target/release/
+    - ./target/release/cargo-contract -V
     - for example in examples/lang2/*; do
-        pushd $example &&
-        cargo contract build &&
+        pushd $example && pwd && echo ${CI_BUILDS_DIR}
+        ${CI_PROJECT_DIR}/target/release/cargo-contract build &&
```

Comment:
If you properly installed `cargo-contract` you normally don't need to access it that way.